include/exiv2/metadatum.hpp

```diff
@@ -80,6 +78,8 @@ class EXIV2API Key {
     @brief Assignment operator. Protected so that it can only be used
            by subclasses but not directly.
    */
+  Key() = default;
```

Comment:
Same comment here about moving the constructors and doxygen doc.

Reply:
Fixed.